dpctl/_sycl_queue.pyx

```diff
@@ -631,34 +631,28 @@ cdef class SyclQueue(_SyclQueue):
         for idx, arg in enumerate(args):
             if isinstance(arg, ctypes.c_char):
                 kargs[idx] = <void*><size_t>(ctypes.addressof(arg))
-                kargty[idx] = _arg_data_type._CHAR
-            elif isinstance(arg, ctypes.c_int):
+                kargty[idx] = _arg_data_type._INT8_T
+            elif isinstance(arg, ctypes.c_uint8):
                 kargs[idx] = <void*><size_t>(ctypes.addressof(arg))
-                kargty[idx] = _arg_data_type._INT
-            elif isinstance(arg, ctypes.c_uint):
+                kargty[idx] = _arg_data_type._UINT8_T
+            elif isinstance(arg, ctypes.c_short):
                 kargs[idx] = <void*><size_t>(ctypes.addressof(arg))
-                kargty[idx] = _arg_data_type._UNSIGNED_INT
-            elif isinstance(arg, ctypes.c_uint8):
+                kargty[idx] = _arg_data_type._INT16_T
+            elif isinstance(arg, ctypes.c_ushort):
```

Comment:
Sorry for being late, but why `if instance(arg, ....)` still uses old style types?